CHANGELOG.md

```diff
@@ -14,6 +14,8 @@ This release will contain consensus-breaking changes.
 - Clarity function `stx-transfer?` now takes a 4th optional argument, which is a memo.
 - Added a new parser which will be used to parse Clarity code beginning with 2.1,
   resolving several bugs in the old parser and improving performance.
+- Documentations will indicate explicitly which Clarity version introduced each
```

Comment:
Grammar: "Documentation" is a collective noun -- it's already plural.

Reply:
lol. ok changed.